choreolib/src/main/java/choreo/auto/AutoChooser.java

```diff
@@ -43,17 +44,20 @@ public class AutoChooser implements Sendable {
   private final HashMap<String, Supplier<Command>> autoRoutines =
       new HashMap<>(Map.of(NONE_NAME, Commands::none));
 
-  // private final StringEntry selected, active;
-  // private final StringArrayEntry options;
+  private int allianceId = -1;
```

Comment:
Why use a number here?

Reply:
For easier comparison

Reply:
What is wrong with using `.equals()`? As per `Optional`'s documentation:
> Indicates whether some other object is "equal to" this Optional. The other object is considered equal if:
    - it is also an Optional and;
    - both instances have no value present or;
    - the present values are "equal to" each other via equals().